src/parser/jobs/mnk/modules/Revolution.tsx

```diff
@@ -0,0 +1,48 @@
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const MIN_POTENCY_LOSS = 150
+const MAX_POTENCY_LOSS = 250
```

Comment:
I think this can be done more dynamically via data. If you add the potency fields to the Blitzes, you can operate on them directly such as `this.data.actions.ELIXIR_FIELD.potency` so updating the data can be done by layers, allowing the code to work for any version of the expansion even if potencies change. Hardcoding like this basically forces you to have per-patch handling or outright not support certain patches if this changes.